src/scripts/plugincatalogpage.js

```diff
@@ -1,78 +1,212 @@
-define(["loading", "libraryMenu", "globalize", "cardStyle", "emby-linkbutton", "emby-checkbox", "emby-select"], function(loading, libraryMenu, globalize) {
+define(["loading", "libraryMenu", "globalize", "cardStyle", "emby-linkbutton", "emby-checkbox", "emby-select"], function (loading, libraryMenu, globalize) {
     "use strict";
 
     function reloadList(page) {
-        loading.show(), query.IsAppStoreSafe = !0;
-        var promise1 = ApiClient.getAvailablePlugins(query),
-            promise2 = ApiClient.getInstalledPlugins();
-        Promise.all([promise1, promise2]).then(function(responses) {
+        loading.show();
+        query.IsAppStoreSafe = true;
+        var promise1 = ApiClient.getAvailablePlugins(query);
+        var promise2 = ApiClient.getInstalledPlugins();
+        Promise.all([promise1, promise2]).then(function (responses) {
             populateList({
                 catalogElement: page.querySelector("#pluginTiles"),
                 noItemsElement: page.querySelector("#noPlugins"),
                 availablePlugins: responses[0],
                 installedPlugins: responses[1]
-            })
-        })
+            });
+        });
     }
 
     function populateList(options) {
-        populateListInternal(options)
+        populateListInternal(options);
     }
 
     function getHeaderText(category) {
         category.replace(" ", "").replace(" ", "");
-        return "Channel" === category ? category = "Channels" : "Theme" === category ? category = "Themes" : "LiveTV" === category ? category = "HeaderLiveTV" : "ScreenSaver" === category && (category = "HeaderScreenSavers"), globalize.translate(category)
+
+        if ("Channel" === category) {
+            category = "Channels";
+        } else {
+            if ("Theme" === category) {
+                category = "Themes";
+            } else {
+                if ("LiveTV" === category) {
+                    category = "HeaderLiveTV";
+                } else {
+                    if ("ScreenSaver" === category) {
+                        category = "HeaderScreenSavers";
+                    }
+                }
+            }
+        }
+
+        return globalize.translate(category);
     }
 
     function isUserInstalledPlugin(plugin) {
-        return -1 === ["02528C96-F727-44D7-BE87-9EEF040758C3", "0277E613-3EC0-4360-A3DE-F8AF0AABB5E9", "4DCB591C-0FA2-4C5D-A7E5-DABE37164C8B"].indexOf(plugin.guid)
+        return -1 === ["02528C96-F727-44D7-BE87-9EEF040758C3", "0277E613-3EC0-4360-A3DE-F8AF0AABB5E9", "4DCB591C-0FA2-4C5D-A7E5-DABE37164C8B"].indexOf(plugin.guid);
```

Comment:
what is this stuff?.. do we even want to keep it?

Reply:
It looks like it's a weird function to check if a plugin is installed by the user or if it was bundled with the server :man_shrugging: I'm guessing we do want something like that (hard to say), but definitely not implemented like this :laughing:

Reply:
emby likes to treat their own bundled plugins specially. In my opinion there should be no difference, a plugin is a plugin so hopefully this can be removed with time? For now I think it would make no difference to replace the thing with "return true"